plugins/reports/api/reports.js

```diff
@@ -640,30 +640,36 @@ var metricProps = {
 
     reports.send = function(report, message, callback) {
         if (report.emails) {
-            for (var i = 0; i < report.emails.length; i++) {
-                var msg = {
+            for (let i = 0; i < report.emails.length; i++) {
+                const msg = {
                     to: report.emails[i],
                     from: versionInfo.title,
                     subject: report.subject,
                     // if report contains customize message for each email address, use reports.messages[i]
                     html: report.messages && report.messages[i] && report.messages[i].html || message,
                 };
 
-                if (report.messages && report.messages[i]) {
-                    msg.list = {
-                        unsubscribe: {
-                            url: report.messages[i].unsubscribeLink,
-                            comment: report.unsubscribe_local_string || 'Unsubscribe'
-                        }
-                    };
-                }
-
-                if (mail.sendPoolMail) {
-                    mail.sendPoolMail(msg);
-                }
-                else {
-                    mail.sendMail(msg);
-                }
+                var options = { "directory": "/tmp", "width": "1028px", height: "1000px" };
+                pdf.create(msg.html, options).toFile('/tmp/email_report_' + new Date().getTime() + '.pdf', function(err, res) {
+                    if (err) {
+                        return console.log(err);
```

Comment:
instead of console.log please use the logger module.
The error message should be displayed on debug `log.d` only.